GoogleDataTransportCCTSupport/GDTCCTWatchOSTestApp/GDTCCTWatchOSTestApp WatchKit Extension/ExtensionDelegate.swift

```diff
@@ -1,10 +1,18 @@
-//
-//  ExtensionDelegate.swift
-//  GDTCCTWatchOSTestApp WatchKit Extension
-//
-//  Created by Doudou Nan on 2/26/20.
-//  Copyright © 2020 Google. All rights reserved.
-//
+/*
```

Comment:
I think you've checked in some test app code to ddnan-gdtwatchos that I haven't reviewed. Seeing this code is unfamiliar, and ideally, all PRs going into your feature branch should be reviewed. Please create a PR for that by reverting your previous commits and cherry picking them onto a branch that you can merge back in.